### PR TITLE
KS-SK1060-WHR-Update-configure_ipv6-for-WEBHOSTING-Translations

### PR DESCRIPTION
**Pull request by Kazarus4**
All translations are done.

Gitpod : 
- links =>all in 200 ok status
- rendering => all seems to be good

Need proofreading for DE, EN, latin and PL versions.

PS => I corrected the order for EN subs guide (3 => 01) => cluster and shared hosting IP adresses (I saw a mistake in this order during my gitpod checking fot ipv6 guides.

### CODE REVIEW

File: pages/web/hosting/clusters_and_shared_hosting_IP/guide.en-asia.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: 'IP address list for Web Hosting clusters'
 slug: list-of-ip-addresses-of-web-hosting-clusters
 excerpt: 'Find out the appropriate IP addresses to use with your OVHcloud Web Hosting plan'
 section: 'Web Hosting configuration'
-order: 3
+order: à&
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/clusters_and_shared_hosting_IP/guide.en-asia.md]: This won't work ;-)

File: pages/web/hosting/configure_ipv6/guide.es-es.md [hunk 2]
```diff
@@ -1,74 +1,83 @@
...
+Como resultado, se introdujo el protocolo **IPv6** para conectar a la red Internet hasta 340 sextillones de dispositivos. Su implementación toma tiempo porque toda la red de Internet debe integrar esta nueva norma. 
 
-Para más información, consulte el artículo de [Wikipedia](https://es.wikipedia.org/wiki/IPv6){.external} sobre el protocolo IPv6.
+Al estar menos disponibles las direcciones IPv4, es más difícil añadir nuevas máquinas a la red de Internet con la norma IPv4. No obstante, las conexiones con una dirección IPv6 solo son útiles si, por ejemplo, su sitio web también está disponible con el mismo protocolo. De este modo, cuanto más sitios web estén disponibles en IPv6, mayor será el número de actores presentes en la red de internet que transferirán sus dispositivos o máquinas a este nuevo protocolo.
 
-## Objetivo
+Para más información, consulte el artículo de [Wikipedia](https://fr.wikipedia.org/wiki/IPv6){.external} sobre el protocolo IPv6.
```
Comment (OVH-Team-Guides) [inline pages/web/hosting/configure_ipv6/guide.es-es.md]: please use the EN wikipedia page for all subs other than FR

File: pages/web/hosting/clusters_and_shared_hosting_IP/guide.en-asia.md (hunk 1 above)
Comment (Kazarus4) [reply pages/web/hosting/clusters_and_shared_hosting_IP/guide.en-asia.md]: corrected

File: pages/web/hosting/configure_ipv6/guide.es-es.md (hunk 2 above)
Comment (Kazarus4) [reply pages/web/hosting/configure_ipv6/guide.es-es.md]: It placed directly subs links for all subs

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**